compiler/luci-interpreter/src/kernels/Softmax.cpp

```diff
@@ -35,7 +36,14 @@ Softmax::Softmax(const Tensor *input, Tensor *output, const SoftmaxParams &param
 
 void Softmax::configure()
 {
-  assert(input()->element_type() == output()->element_type());
+  LUCI_INTERPRETER_CHECK(input()->element_type() == output()->element_type());
+  LUCI_INTERPRETER_CHECK(input()->shape().num_dims() >= 1);
+  if (input()->element_type() == DataType::U8 || input()->element_type() == DataType::S8)
+  {
```

Comment:
Please add `LUCI_INTERPRETER_CHECK(output()->zero_point() == 0)`.

Although there is no assert in TFLite source code, TFLite kernel assumes output zero point 0.

In `optimized_ops.h`,
```cpp
inline int32_t QuantizeSoftmaxOutput(uint8_t* output_data, float prob_rescaled,
                                     int32_t zero_point) {
  return static_cast<int32_t>(prob_rescaled + 0.5);
}
```

Reply:
Okay I will add this!!